src/odemis/driver/hamamatsurx.py

```diff
@@ -497,9 +498,9 @@ def _getDataFromBuffer(self):
                 # can be NoneType if img_num to high!!
                 # TODO looks like img_info can be empty/None type object -> need a check here?
```

Comment:
Just add a check for the img_info, and if it's empty, log a warning. Also leave a `TODO check whether it ever happens`. After a few weeks, you can check the log, and conclude whether you can drop the check or not.